### DISCUSSION TITLE
MONAI & Processing MPMRI Data

### DISCUSSION THREAD

**Discussion by agokli** [Project-MONAI/MONAI#8270] [General]
Is there any MONAI  tutorial or  knowledge base for processing MPMRI Data - DICOM files  (Fluid Attenuated Inversion Recovery (FLAIR),  T1-weighted pre-contrast (T1w),  T1-weighted post-contrast (T1Gd),  T2-weighted (T2)) and multi label?

For an example, such data is used in Kaggle's RSNA-MICCAI Brain Tumor Radiogenomic Classification competition.

**Comment by ShayanShamsi** [comment]
Not an official MONAI tutorial but maybe look at this?
https://www.kaggle.com/code/mikecho/rsna-miccai-monai-ensemble/notebook
